### PR TITLE
Make the guide more general to Kubernetes

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

**Comment by chbatey** [comment]
Master CI has been broken a far back as travis history goes

File: src/main/paradox/index.md [hunk 1]
```diff
@@ -1,43 +1,40 @@
-# Deploying Lightbend applications to OpenShift
+# Deploying Lightbend applications to Kubernetes
```
Comment (patriknw) [inline src/main/paradox/index.md:1]: Lagom

**Comment by patriknw** [comment]
@chbatey a few small comments here and then we can merge

**Comment by patriknw** [comment]
After merge (and verification that it has been published) we should close https://github.com/lightbend/akka-platform-meta/issues/66

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Lightbend OpenShift Deployment guide
+# Lightbend Kubernetes Deployment guide
```
[the remark this answers is not in the document]
Comment (chbatey) [reply README.md:1]: 👍

**Comment by chbatey** [comment]
Travis has not been happy in as long as the history goes back: https://travis-ci.com/github/lightbend/openshift-deployment-guide/branches

**Pull request closed without merging**